addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -90,15 +90,41 @@ $component-active-color: if($component-active-bg, color-contrast($component-acti
 
-$h1-font-size: $font-size-base * $o-theme-h1-font-size-multiplier !default;
-$h2-font-size: $font-size-base * $o-theme-h2-font-size-multiplier !default;
-$h3-font-size: $font-size-base * $o-theme-h3-font-size-multiplier !default;
-$h4-font-size: $font-size-base * $o-theme-h4-font-size-multiplier !default;
-$h5-font-size: $font-size-base * $o-theme-h5-font-size-multiplier !default;
-$h6-font-size: $font-size-base * $o-theme-h6-font-size-multiplier !default;
+$h1-font-size: o-website-value('h1-font-size') or $font-size-base * $o-theme-h1-font-size-multiplier !default;
+$h2-font-size: o-website-value('h2-font-size') or $font-size-base * $o-theme-h2-font-size-multiplier !default;
+$h3-font-size: o-website-value('h3-font-size') or $font-size-base * $o-theme-h3-font-size-multiplier !default;
+$h4-font-size: o-website-value('h4-font-size') or $font-size-base * $o-theme-h4-font-size-multiplier !default;
+$h5-font-size: o-website-value('h5-font-size') or $font-size-base * $o-theme-h5-font-size-multiplier !default;
+$h6-font-size: o-website-value('h6-font-size') or $font-size-base * $o-theme-h6-font-size-multiplier !default;
 
 $headings-font-family: $o-theme-headings-font !default;
+$h2-font-family: $o-theme-h2-font !default;
+$h3-font-family: $o-theme-h3-font !default;
+$h4-font-family: $o-theme-h4-font !default;
+$h5-font-family: $o-theme-h5-font !default;
+$h6-font-family: $o-theme-h6-font !default;
+$headings-line-height: o-website-value('headings-line-height') !default;
+$h2-line-height: o-website-value('h2-line-height') or $headings-line-height !default;
+$h3-line-height: o-website-value('h3-line-height') or $headings-line-height !default;
+$h4-line-height: o-website-value('h4-line-height') or $headings-line-height !default;
+$h5-line-height: o-website-value('h5-line-height') or $headings-line-height !default;
+$h6-line-height: o-website-value('h6-line-height') or $headings-line-height !default;
+$headings-margin-top: o-website-value('headings-margin-top') or 0 !default;
+$h2-margin-top: o-website-value('h2-margin-top') or $headings-margin-top !default;
+$h3-margin-top: o-website-value('h3-margin-top') or $headings-margin-top !default;
+$h4-margin-top: o-website-value('h4-margin-top') or $headings-margin-top !default;
+$h5-margin-top: o-website-value('h5-margin-top') or $headings-margin-top !default;
+$h6-margin-top: o-website-value('h6-margin-top') or $headings-margin-top !default;
+$headings-margin-bottom: o-website-value('headings-margin-bottom') or ($spacer * .5) !default;
+$h2-margin-bottom: o-website-value('h2-margin-bottom') or $headings-margin-bottom !default;
+$h3-margin-bottom: o-website-value('h3-margin-bottom') or $headings-margin-bottom !default;
+$h4-margin-bottom: o-website-value('h4-margin-bottom') or $headings-margin-bottom !default;
+$h5-margin-bottom: o-website-value('h5-margin-bottom') or $headings-margin-bottom !default;
+$h6-margin-bottom: o-website-value('h6-margin-bottom') or $headings-margin-bottom !default;
```

Comment:
Most of these do not look line bootstrap variables either, they should not be in bootstrap_overridden.scss. Or did we already do that for previous variables in the past? Maybe I am overthinking, it's probably ok :+1: Maybe just add a comment on top of each group where there are custom variables?

In any case, should not we do like we do for the color: h3 color, if not explicitly defined, is equal to h2 color, not h1 color. I think it would make sense for these too, no? To check with PDE when he comes back I guess :grimacing:

Reply:
@qsm-odoo  & @bso-odoo ,
IMO, we change the colors present heading behavior:
If I change H2 color, it shouldn't impact H3, H4, ...
However, if I change H1, it should update all children.

Reply:
Ok, confirmed in MP @bso-odoo : 
- We keep the behavior you implemented: setting config(h3), only impacts h3. Setting config(h1), impacts all hx whose config has not been forced already (no reset of the ones that have been forced, like you implemented)
- Do the same for hx colors: do that in its own commit please :pray: 

Thank you :)

Reply:
Pfew :sweat_drops: ... luckily #75680 does not impact text and header colors anymore.